substrate/frame/llm/src/lib.rs

```diff
@@ -507,6 +385,61 @@ pub mod pallet {
 			Self::deposit_event(Event::<T>::Remarked(data));
 			Ok(())
 		}
+
+		/// Force transfer LLM. Can only be called by Courts.
+		///
+		/// - `from`: Account to transfer from.
+		/// - `to`: Account to transfer to.
+		/// - `amount`: Amount to transfer.
+		///
+		/// Emits:
+		/// * `LLMPoliticsLocked`
+		/// * `LLMPoliticsUnlocked`
+		#[pallet::call_index(8)]
+		#[pallet::weight(10_000)] // FIXME
+		pub fn force_transfer(
+			origin: OriginFor<T>,
+			from: LLMAccount<T::AccountId>,
+			to: LLMAccount<T::AccountId>,
+			amount: T::Balance,
+		) -> DispatchResult {
+			let caller: T::AccountId = ensure_signed(origin)?;
+			ensure!(Courts::<T>::get().contains(&caller), Error::<T>::NotCourt);
+
+			let politipool_account = Self::get_llm_politipool_account();
+			let transfer_from = match from {
+				LLMAccount::Liquid(_) => return Err(Error::<T>::InvalidAccount.into()),
+				LLMAccount::Locked(account) => {
+					let politics_balance = LLMPolitics::<T>::get(account.clone());
+					ensure!(politics_balance >= amount, Error::<T>::LowBalance);
+
+					LLMPolitics::<T>::mutate(account.clone(), |b| *b -= amount);
+					Self::deposit_event(Event::<T>::LLMPoliticsUnlocked(account, amount));
+					politipool_account.clone()
+				},
+			};
+			let transfer_to = match to {
+				LLMAccount::Liquid(account) => account,
```

Comment:
This means that Courts are able to instantly unlock staked LLM. Is that OK or do we want only staked -> staked moves?

Reply:
staked -> staked for now

Reply:
Changed :+1: